### DISCUSSION TITLE
With prefetching dynamic segments Is it basically impossible to do soft navigation?

### DISCUSSION THREAD

**Discussion by bairrada97** [vercel/next.js#44758] [Help]
https://outscore-next-hsabhi1fo-bairrada97.vercel.app/europe-berlin/date/2023-01-10 I think it helps if I show an example.

so on the navigation you have days where you can switch between them, this are `<Link /> `. Those pages are dynamic segments and Next doesnt know at build time they exist. if  you open the network tab you see they are being prefetched but when navigating It does hard refresh. this only happens in production and not locally.

```
export const dynamicParams = true;

export async function generateStaticParams() {
  return [];
}
```

using this similar approach on next 12 I could do a soft navigation but it seems on next 13 its not possible based on docs https://beta.nextjs.org/docs/routing/linking-and-navigating#prefetching

Any idea if Im doing it correct or am I missing something?

**Comment by nabildroid** [comment]
any updates on this?
